src/welcome.stories.mdx

```diff
@@ -6,4 +6,22 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 This is the pattern library for Cloud Four.
 
-Details coming soon.
+## Installation
+
+### Download Compiled CSS and JS
+
+Coming soon.
```

Comment:
I don't even know if we want to support this. I'm torn. There's always https://unpkg.com/ if we need it for demos and stuff.

Reply:
That's fair, I'll remove it.